_limitations/set_limitations.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Unsupported SET Statements 
+nav_order: 2
+---
+
+## Unsupported SET statements
+
+The following SET statements are available for use with Babelfish versions 1.2.0 and higher:
+
+- `SET FMTONLY` - `SET FMTONLY ON/OFF` applies only to `SELECT *` queries, and is ignored for all other queries.
```

Comment:
various options are missing like SET QUOTED_IDENTIFIER, SET IDENTITY_INSERT, SET NOCOUNT, SET XACT_ABORT and others

Reply:
Again, we want to document unsupported features instead of supported features...

Reply:
This will be addressed in a subsequent PR to avoid merge conflicts...